zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -6557,6 +6557,12 @@ components:
         parentElementInstanceKey:
           type: string
           description: The parent element instance key.
+        tags:
+          description: list of tags that will be associated with the process instance, and will also be forwarded to job workers etc.
```

Comment:
🔧 Please start with Uppercase, also I would not user etc. . To which element this tags will be forwarded to ? Maybe we should write down the full list. We should be more clear 👍